docs/output.md

```diff
@@ -124,10 +124,36 @@ The resulting FASTQ files will only be present in your results directory if you
   - `*.{bai,csi}`: Index file corresponding to a BAM file which is for faster downstream steps (e.g. SAMtools).
   - `*.flagstat`: Statistics of aligned reads from SAMtools `flagstat`.
 
-  </details>bam,bai,csi
+  </details>
 
   [BWA](https://bio-bwa.sourceforge.net/bwa.shtml) is a software package for mapping low-divergent sequences against a large reference genome, such as the human genome. It consists of three algorithms: BWA-backtrack (a.k.a `bwa aln`), BWA-SW and BWA-MEM. The first algorithm is designed for Illumina sequence reads up to 100bp, while the rest two for longer sequences ranged from 70bp to 1Mbp. BWA-MEM and BWA-SW share similar features such as long-read support and split alignment, but BWA-MEM, which is the latest, is generally recommended for high-quality queries as it is faster and more accurate. BWA-MEM also has better performance than BWA-backtrack for 70-100bp Illumina reads.
 
+### BAM Filtering
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `bam_filtering/`
+
+  - `*length.filtered.bam`: BAM file containing mapped and unmapped reads with length filtered reads.
+  - `*.filtered.bam`: BAM file containing mapped quality filtered reads (and optionally length filtering and unmapped reads, if specified by the user with the corresponding parameters).
+  - `*.{bai,csi}`: Corresponding index files of any generated BAM files.
+  - `*.unmapped_other_{1,2,other,singleton}.fastq.gz`: FASTQ file containing only unmapped reads, if specified for generation, without length filtering.
+    - If you do read merging or have single-end data you will only get one file containing (`*_other*`) containg all reads (the others will be empty).
+    - If you skip merging for _paired-end data_ you'll get reads separately (`_1`, `_2`, `_singleton`), with `other` being empty.
+  - `*mapped_{1,2,other,singleton}.fastq.gz`: FASTQ file containing only mapped reads, if specified for generation, without length or quality filtering. If you do read merging you will only get one file containing (`*_other*`) containg all reads. If you skip merging you'll get them separately (`_1, _2 etc.`).
```

Comment:
```suggestion
  - `*mapped_{1,2,other,singleton}.fastq.gz`: FASTQ file containing only mapped reads, if specified for generation, without length or quality filtering. If you do read merging you will only get one file containing (`*_other*`) containg all reads. If you skip merging you'll get them separately (`_1`, `_2` etc.).
```

Reply:
why 'etc.'? Shouldn't that be specific?

Reply:
No not necessarily, output may also depend if there are singletons or not